README.md

```diff
@@ -43,7 +43,9 @@ This repository contains all information about the **D**iLoc**J**son**S**chedule
     - Unterstützung des Zug-Verbindungstyps 'Durchbinden' hinzugefügt.                  
 - **v2.4.1** 
     - 'train' aus nextTrainNumber und nextTrainLinkType entfernt, weil es redundant ist.
-    - nextPiNumber hinzugefügt                 
+    - nextPiNumber hinzugefügt  
+- **v2.4.2**
+    - timestamp hinzugefügt für neu generiert Jahresfahrplan.
```

Comment:
Don't forget to adjust this if the name is changed.
Also I would suggest "xxx für Erzeugungszeitpunkt des Fahrplans hinzugefügt." because this is not only for the period schedule but also day schedules.